### PR TITLE
Feature/change walkings

### PR DESCRIPTION
**Pull request by AlbertPG**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997&bid=18460865)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+public class ActualWalkingFragment extends Fragment implements OnMapReadyCallback, ActualWalkingCommunication {
+    private static final float ZOOM = 16.0f;
+    private static final int REQUEST_INTERVAL = 10000;
+    private static final int FASTEST_INTERVAL_REQUEST = 5000;
+    private static Context context;
+    private static LayoutInflater inflater;
+    private static String errorMessage;
+    private GoogleMap googleMap;
+    private static FusedLocationProviderClient fusedLocationProviderClient;
+    private static LocationRequest locationRequest;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java:53]: Codacy Issue found: [Static variable definition in wrong order.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    private static LocationCallback locationCallback;
+
+    private FragmentActualWalkingBinding binding;
+    private MapView mapView;
+    private List<LatLng> coordinates;
+    private Polyline actualPolyline;
+
+    @Override
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java:61]: Codacy Issue found: [NCSS for this method is 26 (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+ */
+public class ActualWalkingFragment extends Fragment implements OnMapReadyCallback, ActualWalkingCommunication {
+    private static final float ZOOM = 16.0f;
+    private static final int REQUEST_INTERVAL = 10000;
+    private static final int FASTEST_INTERVAL_REQUEST = 5000;
+    private static Context context;
+    private static LayoutInflater inflater;
+    private static String errorMessage;
+    private GoogleMap googleMap;
+    private static FusedLocationProviderClient fusedLocationProviderClient;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java:52]: Codacy Issue found: [Static variable definition in wrong order.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+    private static LocationRequest locationRequest;
+    private static LocationCallback locationCallback;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java:54]: Codacy Issue found: [Static variable definition in wrong order.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java (hunk 2 above)
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java:61]: Codacy Issue found: [Method length is 32 lines (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997&bid=18460865)

**Review by enrichp — DISMISSED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997&bid=18460865)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997&bid=18460865)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997&bid=18460865)

**Review by xavier-campos-diaz — DISMISSED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997&bid=18460865)

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by enrichp — APPROVED** [review]

**Pull request merged**